src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -621,8 +621,10 @@ AutoExifRotate =
 #    mkv      Matroska Video                                video/x-matroska
 #    mlp      Meridian Lossless Packing                     audio/vnd.dolby.mlp
 #    mov      QuickTime File Format                         video/quicktime
+#    mp2      MPEG Audio Layer II                           audio/mpeg
```

Comment:
@Nadahar I'm not trying to make you difficulties...just that **audio/x-ape** is much more supported and more standard. if you don't trust me just do a Google search and check the protocolInfo of the renderers...